src/main/java/edu/harvard/iq/dataverse/branding/BrandingUtil.java

```diff
@@ -16,10 +16,13 @@
     private static SettingsServiceBean settingsService;
 
     public static String getInstallationBrandName() {
-        
-        String brandName = settingsService.getValueForKey(SettingsServiceBean.Key.InstallationName);
-        //Separate if statement simplifies test setup, otherwise could use the getValueForKey method with a default param
-        if(brandName==null) {
+
+        String brandName = null;
+        if (settingsService != null)
+            brandName = settingsService.getValueForKey(SettingsServiceBean.Key.InstallationName);
+        // Separate if statement simplifies test setup, otherwise could use the
+        // getValueForKey method with a default param
+        if (brandName == null && dataverseService != null) {
```

Comment:
Are these changes still needed now that you put `initBrandingUtilWithRootDataverse()` in each test? If not, please revert; if so, could we remove from her and do the necessary set up in the tests (for cleaner code here)?

Reply:
@scolapasta You are right, I restored the original version (plus some small code formatting).